test/performance/tools/common.sh

```diff
@@ -66,52 +66,50 @@ function setup_user() {
 }
 
 # Get cluster credentials for GKE cluster
+# $1 -> cluster_name, $2 -> cluster_zone
 function get_gke_credentials() {
+  name=$1
+  zone=$2
   echo "Updating cluster with name ${name} in zone ${zone}"
   gcloud container clusters get-credentials ${name} --zone=${zone} --project=${PROJECT_NAME} || abort "Failed to get cluster creds"
 }
 
-# Create a new cluster and install serving components and apply benchmark yamls.
-# $1 -> cluster_name, $2 -> cluster_zone, $3 -> node_count
-function create_new_cluster() {
-  # create a new cluster
-  create_cluster $1 $2 $3 || abort "Failed to create the new cluster $1"
-
-  # create the secret on the new cluster
-  create_secret $1 $2 || abort "Failed to create secrets on the new cluster"
-
-  # Setup user credentials to run on GKE for continous runs.
-  get_gke_credentials
-
-  # update components on the cluster, e.g. serving and istio
-  update_cluster $1 $2 || abort "Failed to update the cluster"
-}
-
-# Update resources installed on the cluster with the up-to-date code.
-# $1 -> cluster_name, $2 -> cluster_zone
-function update_cluster() {
+# Delete all the benchmark resources
+# $1 -> cluster_name
+function delete_benchmark_resources() {
   name=$1
-  zone=$2
-  local version="v0.8.0"  
 
   echo ">> Delete all existing jobs and test resources"
   kubectl delete job --all
-  ko delete -f "${TEST_ROOT_PATH}/$1"
+  ko delete -f "${TEST_ROOT_PATH}/$name"
+}
 
+# Install the eventing resources from the repo
+function install_eventing_resources() {
   pushd .
-  cd ${GOPATH}/src/knative.dev
+  cd ${GOPATH}/src/knative.dev/eventing
 
-  echo ">> Update eventing"
-  kubectl apply --selector knative.dev/crd-install=true \
-  -f eventing/releases/download/$version/release.yaml || abort "Failed to apply eventing CRD's"
+  echo ">> Update eventing core"
+  ko apply --selector knative.dev/crd-install=true \
+  -f eventing/config || abort "Failed to apply eventing CRDs"
 
-  kubectl apply -f eventing/releases/download/$version/release.yaml || abort "Failed to apply eventing sources"
+  ko apply \
+  -f eventing/config || abort "Failed to apply eventing resources"
```

Comment:
you are already inside eventing/ 
This should be `ko apply -f config/`

Ditto for others. Or update L90 to remove eventing